### ISSUE TITLE
Would it add an export function?

### ISSUE THREAD

**Issue by AlexLau0907** [lynchjames/obsidian-mind-map#41]
export to an image or xmind format?

**Comment by alexfreyre** [comment]
it would be amazing if there was a way to export the image with adjustable resolution scale!!!

**Comment by MaksimZinovev** [comment]
Yes, adjustable resolution would be great. Sometimes my mindmaps are very large so after exporting to image, text is not readable

**Comment by canmetan** [comment]
It would be great to have this feature!

**Comment by troilus** [comment]
It would be great to have this feature!
